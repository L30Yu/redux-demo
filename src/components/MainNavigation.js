import React from "react";

export default function MainNavigation() {
  return (
    <nav className="navbar navbar-expand-lg sticky-top navbar-light bg-light">
      <a className="navbar-brand" href="">
        Grocery Co.
      </a>
      <div
        className="collapse navbar-collapse d-flex flex-row-reverse"
        id="navbarNavAltMarkup"
      >
        <div className="navbar-nav">
        </div>
      </div>
    </nav>
  );
}
