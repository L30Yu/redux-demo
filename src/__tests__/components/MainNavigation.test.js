import React from 'react';
import MainNavigation from '../../components/MainNavigation';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<MainNavigation />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
