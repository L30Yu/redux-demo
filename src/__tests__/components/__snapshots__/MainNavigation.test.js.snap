// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navbar navbar-expand-lg sticky-top navbar-light bg-light"
>
  <a
    className="navbar-brand"
    href=""
  >
    Grocery Co.
  </a>
  <div
    className="collapse navbar-collapse d-flex flex-row-reverse"
    id="navbarNavAltMarkup"
  >
    <div
      className="navbar-nav"
    />
  </div>
</nav>
`;
